<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowTimeCourse</class>
 <widget class="QWidget" name="WindowTimeCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
   <property name="spacing">
    <number>8</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame {background: black}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="WidgetTimeCoursePlot" name="widgetPlot" native="true">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>250</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetLegend" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxShowFrameNumber">
       <property name="text">
        <string>Show Frame number on time axis</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Scale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditScale">
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAutoScale">
       <property name="text">
        <string>Auto</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxMaxScale">
       <property name="text">
        <string>Max</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetTimeCoursePlot</class>
   <extends>QWidget</extends>
   <header>WidgetTimeCoursePlot.h</header>
   <container>1</container>
   <slots>
    <slot>SetAutoScale(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxAutoScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>WindowTimeCourse</receiver>
   <slot>OnCheckAutoScale(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxMaxScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>WindowTimeCourse</receiver>
   <slot>OnCheckMaxScale(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>683</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditScale</sender>
   <signal>returnPressed()</signal>
   <receiver>WindowTimeCourse</receiver>
   <slot>OnLineEditScaleReturnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxShowFrameNumber</sender>
   <signal>toggled(bool)</signal>
   <receiver>WindowTimeCourse</receiver>
   <slot>OnCheckShowFrameNumber(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>532</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnLineEditScaleReturnPressed()</slot>
  <slot>OnCheckAutoScale(bool)</slot>
  <slot>OnCheckMaxScale(bool)</slot>
  <slot>OnCheckShowFrameNumber(bool)</slot>
 </slots>
</ui>
